test/LaunchDarkly.JsonStream.Tests/LaunchDarkly.JsonStream.Tests.csproj

```diff
@@ -6,27 +6,35 @@
       SDKs which do not consider "net5.0" to be a valid target framework that can be
       referenced in a project file.
     -->
-    <TestFrameworks Condition="'$(TESTFRAMEWORKS)' == ''">netcoreapp2.1;netcoreapp3.1;net5.0;net452;net461</TestFrameworks>
+    <TestFrameworks Condition="'$(TESTFRAMEWORKS)' == ''">netcoreapp2.0;netcoreapp2.1;netcoreapp3.1;net5.0;net452;net461</TestFrameworks>
     <TargetFrameworks>$(TESTFRAMEWORKS)</TargetFrameworks>
     <RootNamespace>LaunchDarkly.JsonStream</RootNamespace>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Here's an obscure problem I ran into. It turns out that if you omit the `LangVersion` property, it defaults to "latest", which you would think means "the latest version of C# available in this .NET SDK"... and that's _usually_ what it does mean. But in .NET Core 2.0, "latest" means "version 7.0, even though 7.3 is also available". And our code won't build in C# 7.0.